### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by wnerini** [xpollen8/jazzbutcher.com#59] [Show and tell]
> It would get us started
> It would help us all a lot

I think it would help us to organize, if we understood our contributors' skills and any relevant background and had that in one place, instead of all over the welcome thread!

I'll start in...

**Comment by wnerini** [comment]
**Regarding the JBC**
Been a fan since 1989 when a fellow freak turned me onto them ("Hey, you like Bauhaus, right?"). I first saw Pat play 29 years 163 days ago, at the iconic Mason Jar, in Phoenix, Arizona. Pat and the boys lit up that shithole. Sadly, I was a mighty-freaking broke student, who literally sold blood to get the paltry ticket money, so nothing to show for it, but seeing Pat's joy in playing made me want to play music myself and that changed my inner life forever.  Second and last time fate let me see the JBC was the awesome The Great American Music Hall show in '99. A pilgrimage of faith, to see Max & Pat together. Had some small communication with Pat over the years, then overjoyed to find him on social media and not only personally available, but happy to treat a longtime fan as a long-lost friend.


**The Non-JBC**
_Professional Skills:_ 30-year dev, project manager, software architect. Everything from 8-Bit assembly to C game coding, to PHP, to the MS Stack. Current core skills are architecting/leading development on high-availability progressive-HTML applications. Stacks are primarily LAMP, .NET/Core, though Perl and C in the earliest days. C & C++ in the 16 bit era (Amiga, largely). Do A LOT of Db and data transform. Name it and I'll learn it.

_Non-Professional Skills:_ I'm an awful musician (really, I'm not being humble), but I'm game to try anything. I have a small chunk of "for fun" experience with computer-based audio and video editing ([see here](https://www.youtube.com/watch?v=Xie78fIKcWY)). I've run an illegal [bar/music hall/art venue](https://www.facebook.com/photo/?fbid=10152099061181130&set=a.10152099065711130) in Tempe AZ.  I'm an autodidact with some silly papers and a massively buff ability to do research, and Master of Google-Fu. I'm a fumble-fingered typist who leave typos behind him, watch out! I'm also a giant film-nerd with a 24TB horde of both the very best and very worst of cinema, and as a chaser, all of only the most awful Television you can imagine (every episode of The Human Jungle, anyone? Suzanne Sommers in "She's The Sheriff"?). 

So, David and gang, I'm here to help,  and damn happy to be able to contribute!

**Comment by hischeapmoves** [comment]
I've been a JBC fan since the music director at the college radio station I'd just joined introduced me to _Big Planet, Scary Planet_. Side note: she later went on to work for Sky Records in Atlanta and was there when they released _Condition Blue_, I have a couple of nice posters from that. I never got to see a live performance in person, I'm not sure how that happened.

I went to school in Melbourne, Florida (near the Kennedy Space Center) for Computer Science, and started a music magazine shortly before graduation in 1991. _Ink 19_ began in Central Florida but eventually expanded into all of Florida and parts of Georgia by the time it stopped having a printed edition at the end of 2000. It continues publishing online.

The print magazine was always a bootstrap operation with a tiny budget, and we leveraged technology however we could, soliciting articles via newsgroups in 1991, having a remote art director in 1993 that sent PageMaker files over Z-modem at 2am, publishing a HyperCard version on floppy discs in 1995, and so on.

ink19.com has been an active web site since 1996, and it's gone through many generations of content management systems and content deployment strategies in that time. There's about 30,000 articles in the archive, and I'm here partly because I may have some experience I hope I can contribute, but also because there's a lot I think I can learn from this project.

**Reply by xpollen8** [reply]
| Senior Intern: Robert Koss

heh.

I'm curious what, roughly, the ink19 tech stack looks like for delivering the website.

**Reply by wnerini** [reply]
> ... HyperCard version on floppy discs in 1995, and so on.
> 
Been in the wars, have we? Churned out Hypercard stacks, for corporate training, for like 2 years myself!

> ink19.com has been an active web site since 1996, and it's gone through many generations of content management systems and content deployment strategies in that time. There's about 30,000 articles in the archive, and I'm here partly because I may have some experience I hope I can contribute, but also because there's a lot I think I can learn from this project.

Awesome, this experience will definitely  be helpful and I hope to learn from this project as well!

**Reply by hischeapmoves** [reply]
> | Senior Intern: Robert Koss

Bob is a systems engineer at Northrop Grumman these days...

> I'm curious what, roughly, the ink19 tech stack looks like for delivering the website.

The current stack has a Rails-based back end for the content store, a vue.js-based front end for the CMS, and Wordpress for delivering the content (the Rails back end uses wp-cli to write articles to WP). I'm in the process of ditching WP for a vue.js based front end, hopefully get that done by the end of the year.

I have another side project that uses Jekyll ... this is the YAML + markup approach I was talking about earlier: [https://the.lacking.org/](https://the.lacking.org/) | [Repo](https://github.com/ubikorp/lacking.org)

**Reply by hischeapmoves** [reply]
> Been in the wars, have we? Churned out Hypercard stacks, for corporate training, for like 2 years myself!

Despite all its efforts to make me hate it, I still love HyperCard. It was a really amazing toolkit.

![ink19-hypercard](https://user-images.githubusercontent.com/17223/142077555-db4e618b-bc4c-4876-a2e3-0de7f4c08453.jpeg)

**Reply by xpollen8** [reply]
lacking.org. Nice. And far more ambitious than my own [inconvenient.com](http://inconvenient.com) and [virtuallyfriendless.com](http://virtuallyfriendless.com)

**Reply by wnerini** [reply]
> It was a really amazing toolkit.

I loved Hypercard. It funded my transition from multimedia and game development (8/16 bit) to Web and app development (through hardware purchases) in early 90s. It also influenced all my early HTML design. Trivia: All my Hypercard development was done using a Mac emulator running on a 040 Amiga.

I made a living off Wordpress development in 2004-2010, so I know that pain for sure. These days I just work off an Angular client stack with restful API.

**Reply by xpollen8** [reply]
I've been keeping EIder's wordpress-based site alive for a couple years.  (I didn't put it together, but I've been hosting it and trying to keep it running).  Pain.  Yeah, that's the word for it.

**Comment by hischeapmoves** [comment]
Are we the tech team then? Any others?

**Reply by xpollen8** [reply]
3 curmudeons it is then!

**Comment by indieplumber** [comment]
Hello, I am just about to hit 50. First became aware of JB via the Spacemen 3.
A musician myself, i hung up my guitar aged 21 and concentrated on studies that via sociology ended up with me being registered fro a PhD . Now, this is where it gets fishy. My proposal was concerned with popular music history - but due to the nature of the beast my ACTUAL project became - Researching these people who call themselves researchers but are actually bullying leaches. 
Then i watched my dad die. 
I fuked off the PhD in the way that would cause the maximum damage for the department. It did damage to me and i have been out of work for a long time. I managed 2 days in a call centre ....
I am published in the field with a piece about what music heritage practitioners laughingly like to call Oral History but is as far as they are concerned a way of drawing down an extra 100k and not have to worry about the outputs. 
I remain very angry at the state of music heritage practice. The people are horrible and the stuff they produce is frivolous and pathetic.

I have coped wit my problems by writing songs. I write operas. 

Check out the journey i have been on at [www.heavytourism.com](url)

I have no coding skills but have uploaded much content to webpages in the hope of digital archiving music ephemera. I am a very well experienced heritage practitioner and 10 years ago was all set to be someone in the academy. But the academy can f right off forever. The academy is the problem. Its the biggest gang in town. I love doing cut n paste graphics and my fanzines change hands on ebay for double figure sums - they   50p when released. I have had problems on internet forums in the past. I hope that this community will forgive my lack in protocol knowledge.

I never saw Pat live but friends of his stayed in my house. Many many musicians have stayed in my house over the years. 

Hope that gives you a good idea of my position. The retoric of digital preservation i have gritted my teeth through in various university meeting rooms is not merely a way for failed arts history graduates to endure they keep up with their private schooled peers .... I'll stop now

**Comment by FaustFlo** [comment]
Hello guys,
My name is Florence Dolisi, but in the 80s and 90s, when I was doing music and radio, my nickname was "Faustine". That's how Pat Fish used to call me. I'm French, I'm 55 years old and when I bought my first JBC records in 1985, I was still in high school. I saw the JBC on stage several times in 87 and 88, and once in 94. Like all of you, I love their music and I loved the guy, so sweet and gentle. He was a class act. 
Since the 90s, I have become a translator from English to French and from Spanish to French. I translate mainly science fiction novels, a genre I'm passionate about, and also books on scientific subjects. 
So if you agree, I will translate from French to English the interviews of Pat Fish published in the French press. My translations won't be perfect but at least, they'll be better than the ones from Google !
Unfortunately, I can't help you with the maintenance of the site, as I'm pretty bad at informatics. I can't be part of the tech team, sorry...

**Reply by xpollen8** [reply]
Thanks for dropping in, Flo!

I've sent you a collaborator invite.

If you have things to upload (images, etc), you can use the custom [JBC Uploader](https://upload.jazzbutcher.com) which helps track things better.

Translations: there are several robotic French-language items already in the ["press" section](http://jazzbutcher.com/press).  Feel free to [open a new issue](https://github.com/xpollen8/jazzbutcher.com/issues) for each article you might work on, and then just drop you translation as a comment on the issue, and I can take it from there.

Thank you for joining.
